### ISSUE TITLE
[Feature Request]: Support kebab-case type hinting for Blade component properties

### ISSUE THREAD

**Issue by simensen** [laravel-idea/plugin#469]
### Feature Description

Given this Blade component:

```blade
@props(['vcsConnection'])
<div {{ $attributes }}>
    @if($vcsConnection->avatar_url)
        <span class="block relative">
            <img class="h-12 w-12" src="{{ $vcsConnection->avatar_url }}" alt="{{ $vcsConnection->display_name }}">
            @svg('simpleicon-' . $vcsConnection->icon, 'h-4 w-4 mb-0.5 mr-0.5 absolute bottom-0 right-0 rounded-full bg-white')
        </span>
    @else
        @svg('simpleicon-' . $vcsConnection->icon, 'w-8 h-8 sm:w-4 sm:h-4 md:w-12 md:h-12')
    @endif
</div>
```

... both of the following usages of the component work the same way:

```blade
<x-vcs-connection-avatar class="flex-shrink-0" :vcsConnection="$vcsConnection" />
<x-vcs-connection-avatar class="flex-shrink-0" :vcs-connection="$vcsConnection" />
```

... however, Laravel IDEA only includes the camelCase version (`vcsConnection`) as a type hint.

![image](https://user-images.githubusercontent.com/191200/145691588-3a7dab32-05f5-48b9-a76a-aca2e3de448d.png)

[From the current docs](https://laravel.com/docs/8.x/blade#casing) (emphasis added by me):

> Component constructor arguments should be specified using camelCase, while **kebab-case should be used when referencing the argument names in your HTML attributes**. For example, given the following component constructor...

While not directly related to anonymous Blade components like I'm using, I *believe* the intent is that HTML attributes should favor kebab-case over camelCase in similar projects due to the case-insensitive nature of HTML/attributes.

----

If this is something I could help move along by contributing code, I'd be happy to take a crack at if I could get a few pointers on where to start. :)

Thanks!

**Comment by adelf** [comment]
Hi, Beau. I have this option for livewire components, but not for Blade components :) Strange. I'll add it. And probably will ask Twitter about the preferred default value. Thanks!

**Comment by adelf** [comment]
Fixed in 4.4.6
